src/poker.js

```diff
@@ -1,35 +1,108 @@
-// Input: 2 arrays of strings, each with 2 iteams
-// Output: One array
-
-// Check if both iteams in array are the same
-// If they are, it's a pair
-// If not, it's not a pair
-// If neither of the arrays have pairs, return []
-// If one of the arrays has a pair, return that pair
-
-// If both of the arrays has pairs, figure out which pair scores higher
-// Check every card in an array against some kind of point value
-// Do that for both arrays, compare the total score
-// then return the array with the highest score
-
-// const pointMap = {
-//   Card2: 2,
-//   Card3: 3,
-//   Card4: 4,
-//   Card5: 5,
-//   Card6: 6,
-//   Card7: 7,
-//   Card8: 8,
-//   Card9: 9,
-//   Card10: 10,
-//   CardJack: 11,
-//   CardQueen: 12,
-//   CardKing: 13,
-//   CardAce: 14
-// }
+const pointMap = {
+  2: 2, 3: 3, 4: 4, 5: 5, 6: 6, 7: 7, 8: 8, 9: 9, 10: 10, J: 11, Q: 12, K: 13, A: 14
+}
+// 1) Indentation
+// 2) If statement for loops, functions need {} before the code inside them
+// 3) Don't worry about lots of functions. Get code working then extract to other functions after
 
 class Poker {
-  // Write your implementation here
+  winningPair (handOne, handTwo) {
+  // cardHands: ['A', 'A'], ['3', 'K']
+    const handOneIsPair = this.isPairCheck(handOne) // ['A', 'A'] = true, because is pair
+    const handTwoIsPair = this.isPairCheck(handTwo) // ['3', 'K'] = false, because is not a pair
+    console.log('handOne = ', handOne, handOneIsPair)
+    console.log('handTwo = ', handTwo, handTwoIsPair)
+
+    // if handOne && handTwo are both false return []
+
+    if (!handOneIsPair && !handTwoIsPair) { return [] }
+
+    if (handOneIsPair && !handTwoIsPair) { return handOne }
+
+    if (!handOneIsPair && handTwoIsPair) { return handTwo }
+
+    if (pointMap[handOne[0]] > pointMap[handTwo[0]]) {
+      return handOne
+    } else {
+      return handTwo
+    }
+  }
```

Comment:
Lovely conditional flow to find the winning pair. Especially the exclamation to reverse the value